tests/cypress/package.json

```diff
@@ -34,6 +33,7 @@
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^6.2.0",
     "@typescript-eslint/parser": "^6.2.0",
+    "cypress": "^13.6.4",
```

Comment:
Please move back to `dependencies:` - otherwise `--dev` might be needed for `npm install`

Reply:
thanks .done